WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -914,6 +925,24 @@ private function validate_prefixes() {
 				continue;
 			}
 
+			if ( function_exists( 'iconv_strlen' ) && iconv_strlen( $prefix, $this->phpcsFile->config->encoding ) < self::MIN_PREFIX_LENGTH ) {
```

Comment:
@NielsdeBlaauw What do you think: could the `if` and the `elseif` conditions be combined to get rid of the duplicate error block code ?

Also: the plain `strlen()` function does not take an `$encoding` parameter.